libs/shared/sentry/src/lib/reporting.ts

```diff
@@ -41,6 +41,16 @@ export function tagFxaName(data: any, name?: string) {
   return data;
 }
 
+/** Indicates if error should be sent to Sentry */
+export function ignoreError(err: any): boolean {
```

Comment:
nit: I kinda like the deleted comment/description above. It helped give some context on why they were not reported.